elastic/kibana
# [ObsUX] [Infra] Add missing metrics to Docker container view

Closes https://github.com/elastic/kibana/issues/183354

Metric section
![image](https://github.com/elastic/kibana/assets/31922082/0d85da8b-a651-4557-ac09-e3eee2573c42)

How to test
- The feature is under a FF, on inventory page go to settings and enable Container view
- In containers inventory, select a docker container, you find one, filter by any `docker.` field. Click on a container.
- Container details page should be shown Network and DiskIO charts, as well as CPU and Memory, on Metrics section
